src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/nonassociations/NonAssociationsResourceIntTest.kt

```diff
@@ -682,12 +697,84 @@ class NonAssociationsResourceIntTest : IntegrationTestBase() {
     @Test
     fun `non-association does not exist`() {
       webTestClient.put()
-        .uri("/non-associations/offender/{offenderNo}/ns-offender/{nsOffenderNo}/close", "A1234AA", "A1234AA")
+        .uri(
+          "/non-associations/offender/{offenderNo}/ns-offender/{nsOffenderNo}/sequence/{typeSequence}/close",
+          "A1234AA",
+          "A1234AA",
```

Comment:
Should we be capturing this offender/nsOffender combination as a badRequest (as they can't be the same)

Reply:
Could do, ok, will add that next time